customskill/examples/http.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"net/http"
+
+	"github.com/mikeflynn/go-alexa/customskill"
+	"github.com/mikeflynn/go-alexa/customskill/request"
+	"github.com/mikeflynn/go-alexa/customskill/response"
+)
+
+var (
+	s customskill.Skill
```

Comment:
This could be a one-line `var s customskill.Skill`

Reply:
do one letter variable names pass golint? I know in python its an issue I'm not sure for Go though.

Reply:
I have not been running golint. I will work on getting that setup

Reply:
got golint up and running and it didn't seem to complain about this.